src/MultiFloats.jl

```diff
@@ -495,10 +495,19 @@ function Base.log(x::MultiFloat{T,N}) where {T,N}
     return y
 end
 
+function Base.log1p(x::MultiFloat{T,N}) where {T,N}
+    y = x+1
+    return Base.log(y)
```

Comment:
This should be `2*atanh(x/(2+x))` for better accuracy and similar speed.

Reply:
Done